docs/tutorials/Biometric-Liveness-Verification/Readme.md

```diff
@@ -0,0 +1,364 @@
+# Biometric Liveness Verification
+
+<!-- AUTHOR romanstrobl 2024-09-26T00:00:00Z -->
+<!-- SIDEBAR _Sidebar.md sticky -->
+<!-- TEMPLATE tutorial -->
+
+This tutorial explains how Wultra's Biometric Liveness Verification components can be deployed, configured and integrated into your application. This solution enables to verify users using advanced biometric verification techniques against their static photos, such as passport photos.
+
+## Introduction
+
+Wultra's Biometric Liveness Verification solution consists of two components which require to be deployed. The components are available as Docker images in Wultra JFrog Artifactory.
+
+### User Data Store
+
+The component can be used as a generic secure storage for any document type. When used within the Biometric Liveness Verification solution, User Data Store is used to store securely user photos. You will not need to integrate with this component, it's used within the Liveness Check Proxy component as a secure storage. However, the component will be used to upload or import user profile photos which serve a basis for the liveness verification. 
+
+### Liveness Check Proxy
+
+The component orchestrates the biometric liveness check and provides a simple REST API for integration with your application. The Liveness Check Proxy uses the User Data Store to retrieve user photos when performing the biometric verification. Furthermore, the component provides an audit trail of performed liveness checks.
+
+## Deploying Backend Components
+
+Both components can be easily deployed using Docker. Use the following commands to download the Docker images:
+
+User Data Store:
+```shell
+docker pull wultra.jfrog.io/wultra-docker/user-data-store:1.3.0-a2871e2f8d033dd4d00bf1a39456fc503be79194
+```
+
+```shell
+docker pull wultra.jfrog.io/wultra-docker/liveness-check-proxy:0.1.0-SNAPSHOT-2024.06.26-0f5031f9d5ca260aa40c28305ddb2e93a7f4e74e
+```
+
+TODO: use the release version of Docker image
+
+You can use one of the supported databases:
+- PostgreSQL 9.5.4 or newer
+- Oracle Database 11g, 12c, 19c, or 21c
```

Comment:
19c - we support long term releases still supported with Extended Support / Unlimited Licence Agreement
https://support.oracle.com/knowledge/Oracle%20Database%20Products/742060_1.html